### ISSUE TITLE
Add possibility to toogle labels

### ISSUE THREAD

**Issue by SiXoS** [google/google-visualization-issues#2004]
It would be great if the possibility to choose which labels you want to display existed. For example in a line graph, you could ctrl click a label to hide that line and then click again to display it.

**Comment by nbering** [comment]
I'm not affiliated with this project but watch it closely because I have some investment in using the charts.  To be clear, is this what you're looking for?

http://stackoverflow.com/questions/17444586/show-hide-lines-data-in-google-chart

Are you looking to hide a label (text describing some data or element of the chart), or a series (the graphed data)?  If you're trying to hide data there's a few solutions out there.  It is a common problem and the solution is a bit of a hack.  I would speak up in support of toggling series data in charts.  When looking for the solution myself I came across enough questions about it to suggest that it's a desired feature.

**Comment by SiXoS** [comment]
Yes, I'm looking for the same as in the stackoverflow. I solved it myself by having a multiselect next to the graph and using that to show/display data lines but a good way to do it by labels would be preffered. If you click on a label it should be grayed out and the respective line dissapear. The graph should also be redrawn as removing a line may make it possible to see the others in greater detail.

**Comment by nbering** [comment]
I have a blog post on how to do all this, but it's specific to an AngularJS setup with [Angular-Google-Chart](https://github.com/angular-google-chart/angular-google-chart). My blog post is here: http://nicholasbering.ca/angularjs/2014/08/16/agc-hide-series/

Bascially, you're replacing the series with a calculated series and then returning null from the calculation.  Then you override the series colour with grey so that the legend indicates that the series is hidden.  Backing up the colour before changing to grey, or having a back up set already stored from the initialization process allows you to restore the colour later.  In my case with Angular-Google-Chart, the AngularJS directive is watching the chart data and configuration for changes and automatically redraws the chart.

If this were a built in feature it would be nice to just have a something like this:

``` JavaScript
var options = {
  series: {
    hidden: true,
    hiddenColor: '#55555'
  },
  legend: {
    showHidden: true
  }
};
```

Just as an off-the-cuff example.  There are many ways this could be handled.
